recipes/textnets/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "textnets" %}
+{% set version = "0.4.10" %}
+{% set hash = "c9c917a76ee02fe34ee4dbf41428a1df26a07430234f578f45dc9e8a8cb13c79" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - textnets = textnets.cli:main
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - Click >=7.0
+    - pandas ==1.0.4
+    - cairocffi ==1.1.0
+    - python-igraph ==0.8.2
+    - spacy ==2.3.0
+    - scipy ==1.4.1
+    - toolz ==0.10.0
+    - leidenalg ==0.8.1
```

Comment:
Do these really need to be this restrictive?

Reply:
I suspect not, I just used the same versions pinned in `setup.py`. I can try to turn these into looser constraints.

Reply:
2a06bbf makes these far less restrictive. Let me know if that looks better to you.

Reply:
Yup that looks good, I'll merge when the CI passes. Thanks! 🍰